Makefile

```diff
@@ -240,6 +254,7 @@ clean: clean-deprecated
 	rm -f bin/postgres-operator
 	rm -f config/rbac/role.yaml
 	[ ! -d testing/kuttl/generated ] || rm -r testing/kuttl/generated
+	[ ! -d testing/kuttl/generated-other ] || rm -r testing/kuttl/generated-other
```

Comment:
❓ Should these two lines be `testing/kuttl/e2e-generated` and `testing/kuttl/e2e-generated-other`?

Reply:
Yes, that looks like an oversight. Thanks  for the catch, updated.